CHANGELOG.md

```diff
@@ -0,0 +1,458 @@
+
+### Performance
+
+ - <csr-id-5d069a362ea70fe81fdb4955801ce4b701705ca1/> lazily compress RistrettoPublicKey
+   Implement lazy compression for RistrettoPublicKey using thread-safe [`OnceCell`](https://docs.rs/once_cell/latest/once_cell/index.html) (currently unstable in std) 
+   
+   ![image](https://user-images.githubusercontent.com/1057902/148182436-1a29d610-4727-49be-9339-f83b53b568ae.png)
+   
+   Red is on current main branch, blue are the changes in the PR
+   
+   - Minor optimisation for lexicographical public key ordering: using reference to the underlying compressed array rather than copying the array onto the stack and then creating a reference
+
+### Commit Statistics
+
+<csr-read-only-do-not-edit/>
+
+ - 3 commits contributed to the release.
+ - 1 day passed between releases.
+ - 3 commits where understood as [conventional](https://www.conventionalcommits.org).
+ - 2 unique issues were worked on: [#81](https://github.comgit//sdbondi/tari-crypto/issues/81), [#86](https://github.comgit//sdbondi/tari-crypto/issues/86)
+
+### Commit Details
+
+<csr-read-only-do-not-edit/>
+
+<details><summary>view details</summary>
+
+ * **[#81](https://github.comgit//sdbondi/tari-crypto/issues/81)**
+    - remove tari script ([`5fd06b7`](https://github.comgit//sdbondi/tari-crypto/commit/5fd06b7fe407ce720759d377265905cf719371dd))
+ * **[#86](https://github.comgit//sdbondi/tari-crypto/issues/86)**
+    - add wasm build artifacts and test run ([`e796bea`](https://github.comgit//sdbondi/tari-crypto/commit/e796bea54ab8c27f1f47ccc36a3401d63f64f1f5))
+ * **Uncategorized**
+    - use standard-version ([`53a0dbd`](https://github.comgit//sdbondi/tari-crypto/commit/53a0dbdfa701f6037b59f671b8238bf1390ba69f))
+</details>
+
+## v0.12.0 (2022-01-18)
+
+### New Features
+
+ - <csr-id-a5defb77e4e212ca5f3fe5e8a1bfe57810efd83e/> add pull request title check
+   This PR adds a lint check for pull request titles to ensure they conform to the conventional commits specification
+ - <csr-id-f608ccfe96c249c63ba91d077145325a47d0aa4d/> add CheckMultiSig and CheckMultiSigVerify
+
+
+### chore (BREAKING)
+
+ - <csr-id-09cc52787272ced3a1a8c9f2edc1e0221f9d8faa/> update rust toolchain (proc_macro_is_available feature)
+
+### Commit Statistics
+
+<csr-read-only-do-not-edit/>
+
+ - 9 commits contributed to the release over the course of 119 calendar days.
+ - 134 days passed between releases.
+ - 5 commits where understood as [conventional](https://www.conventionalcommits.org).
+ - 7 unique issues were worked on: [#58](https://github.comgit//sdbondi/tari-crypto/issues/58), [#59](https://github.comgit//sdbondi/tari-crypto/issues/59), [#60](https://github.comgit//sdbondi/tari-crypto/issues/60), [#61](https://github.comgit//sdbondi/tari-crypto/issues/61), [#62](https://github.comgit//sdbondi/tari-crypto/issues/62), [#68](https://github.comgit//sdbondi/tari-crypto/issues/68), [#74](https://github.comgit//sdbondi/tari-crypto/issues/74)
```

Comment:
I'd be surprised if these links worked?

Reply:
yeah I think the owner of the repo would have to run this